src/analyze/analyze-security.c

```diff
@@ -1910,7 +1910,7 @@ static int property_read_ip_filters(
         if (streq(member, "IPIngressFilterPath"))
                 info->ip_filters_custom_ingress = !strv_isempty(l);
         else if (streq(member, "IPEgressFilterPath"))
-                info->ip_filters_custom_ingress = !strv_isempty(l);
+                info->ip_filters_custom_egress = !strv_isempty(l);
 
         return 0;
 }
```

Comment:
i cherry picked this fix already into git main, given this is an obvious bugfix unlike the rest of the PR, and we don't have to wait for the rest for this.